### DISCUSSION TITLE
Wrong parameters error triggered during some scheduled Velero backups using Minio (on-premises Kubernetes cluster)

### DISCUSSION THREAD

**Discussion by ehemmerlin** [vmware-tanzu/velero#6230] [Community support Q&A]
We use Velero with Minio and Restic on Kubernetes clusters on-premises. After scheduling daily backups we get sometimes a few that triggers the error below related to some wrong parameters. This happens only sometimes on our daily backups: some backup have this error, some other does not have it. Of course, the parameters stay the same for all our scheduled backups. 

`level=info msg="1 errors encountered backup up item" backup=velero/schedule-daily-20230504131129 logSource="pkg/backup/backup.go:431" name=cassandra-0
time="2023-05-04T13:11:39Z" level=error msg="Error backing up item" backup=velero/schedule-daily-20230504131129 error="wrong parameters, namespace \"acme\", backup storage location \"\", repository type \"restic\"" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/repository/ensurer.go:56" error.function="github.com/vmware-tanzu/velero/pkg/repository.(*RepositoryEnsurer).EnsureRepo" logSource="pkg/backup/backup.go:435" name=cassandra-0`

As a side note, we never face this issue using Velero on Azure, only when using Velero with Minio and Restic on Kubernetes clusters on-premises.

Do you know what could be the issue and how to fix it ?

**Comment by bumarcell** [comment]
We're getting the same issue sometimes
